docker-compose.yml

```diff
@@ -8,17 +8,15 @@ services:
         - http_proxy
         - https_proxy
         - no_proxy
-    command: --wait-for-database 30
     image: docker.artifactory.reform.hmcts.net/reform/job-scheduler-api
     container_name: job-scheduler-api
     environment:
       - S2S_URL
       - S2S_SECRET
-      - S2S_NAME
-      - S2S_TTL
```

Comment:
why were those removed??

Reply:
perhaps because not used? was within same cherry-pick really

Reply:
nothing to do with DB

Reply:
Are you sure? If secret is needed then so is name.
Also, let's keep this PR about 'Bug: DB setup'